### ISSUE TITLE
Is distance matrix already normalized?

### ISSUE THREAD

**Issue by margulies** [margulies/topography#4]
if so, do we have the scaling factor for each individual?

**Comment by soligschlager** [comment]
the distance matrix is normalized within the script. It's scaled to a range between 0 and 1 using sklearn.preprocessing.MinMaxScaler. The scaling factor is stdev \* (max - min ) + min. Should this be saved for each individual?

**Comment by margulies** [comment]
It's possible to use un-normed dist, I think that would be more approriate for the case. Just comment out the line where its normed.
